201-2-vms-internal-load-balancer/azuredeploy.json

```diff
@@ -0,0 +1,321 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "subscriptionId": {
+            "type": "string",
+            "metadata": {
+                "Description": "This is your subscriptionId"
+            }
+        },
+        "availabilitySetName": {
+            "type": "string",
+            "metadata": {
+                "Description": "This is the name of the Availability Set"
```

Comment:
For a 201 starter, we don't need to have this as a parameter. Just hardcode it somewhere.